tests/etc/responses.yml.example

```diff
@@ -223,22 +223,6 @@ keymile_nos:
   interface_ip: 1.2.3.5 
   version_banner: "NOS version 2.09 #0001"
   multiple_line_output: "Interface br328"
-  
-dlink_ds:
-  base_prompt: DGS-3120-24TC:admin
-  router_prompt: DGS-3120-24TC:admin#
-  enable_prompt: DGS-3120-24TC:admin#
-  version_banner: "D-Link Corporation"
-  multiple_line_output: "End of configuration file"
-  interface_ip: 192.168.50.10
-
-dlink_ds_telnet:
-  base_prompt: DGS-3120-24TC:admin
-  router_prompt: DGS-3120-24TC:admin#
-  enable_prompt: DGS-3120-24TC:admin#
-  version_banner: "D-Link Corporation"
-  multiple_line_output: "End of configuration file"
-  interface_ip: 192.168.50.10
```

Comment:
Rebase to fix this accidental removal.

Reply:
Please check my new commit, should of fixed it